docs/ref/contrib/admin/index.txt

```diff
@@ -2829,6 +2835,45 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _overriding-default-adminsite:
+
+Overriding default ``AdminSite``
+--------------------------------
+
+.. versionadded:: 1.11
+
+You can override the default `django.contrib.admin.sites.AdminSite` instance with your own:
+just write your own `AdminConfig` and set the `SimpleAdminConfig.default_site`
```

Comment:
"set the `default_site` attribute to the dotted import path of your custom `AdminSite` subclass" ?